openshift/template.yaml

```diff
@@ -0,0 +1,251 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: configuration-anomaly-detection-template
+parameters: []
+objects:
+- apiVersion: tekton.dev/v1beta1
+  kind: PipelineRun
```

Comment:
what does this do?

Reply:
if you mean the PipelineRun. it's for testing purposes, we should move it out as well (as it causes and error when you apply it)

Reply:
This is a test PipelineRun and should not be included.